### ISSUE TITLE
Header - Implement inline SVG

### ISSUE THREAD

**Issue by greenc05** [bbc/simorgh#368]
**Is your feature request related to a problem? Please describe.**
Currently the header component has a text link, this needs replacing with an SVG and this content should be accessible

**Describe the solution you'd like - For accessiblity**
- Add an attribute of focusable="false" to the svg - Without this, when using Jaws with IE and navigating via the ‘tab’ key, the SVG content is read out twice to screen reader users. This is due to a bug with IE in which it makes all SVGs within links/buttons focusable, resulting in you being able to tab to the link and also to the SVG inside the link, this is also an issue for sighted users using the keyboard to navigate with the tab key who will have to tab twice when encountering a link with an svg inside.
- Add an attribute of aria-hidden="true" to the SVG (title is not reliably supported)
- Use a ‘visually hidden’ (also known as off-screen text) span for the textual equivalent of the SVG
- Even if you are following these recommendations, always thoroughly test your exact implementation

```
<a href="https://www.bbc.co.uk/news">
    <svg focusable="false" aria-hidden="true” …>...</svg>
    <span class="gs-u-vh">BBC News</span>
</a>
```

**Additional context**
***High contrast mode on PC***
It's important that the brand logo SVG works with default and custom colour themes, ensure you test this in development.
